src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,63 @@
+package calculator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+class StringCalculatorTest {
+    private StringCalculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new StringCalculator();
+    }
+
+    @DisplayName(value = "빈 문자열 또는 null 값을 입력할 경우 0을 반환해야 한다.")
+    @ParameterizedTest
+    @NullAndEmptySource
+    void emptyOrNull(final String text) {
+        assertThat(calculator.add(text)).isZero();
```

Comment:
지금은 간단한 구조라서 지금처럼 테스트 코드를 작성해도 읽기 쉽겠지만, 테스트 코드가 복잡해지고 또 갯수가 많아지면 테스트 코드를 읽는데 많은 시간이 들 수 있어요. 

가급적 테스트 코드를 given, when, then 의 포맷에 맞춰서 작성해보면 어떨까요? 🙂 (주석으로 given, when, then 해놓은 부분은 개인 선호도 차이니 취사선택하시면 될 것 같아요. 😉)
```suggestion
        // when 
        int actual = calculator.add(text); 
        
        // then
        assertThat(actual).isZero();
```